asmcomp/mach.mli

```diff
@@ -43,6 +43,10 @@ type operation =
   | Icall_imm of string
   | Itailcall_ind
   | Itailcall_imm of string
+  | Iperform
+  | Itail_delegate
```

Comment:
`delegate` is always at tail position. So should `Itail_delegate` be just `Idelegate`? Even otherwise, `Itaildelegate` is more consistent with the naming convention for calls.